gpkit/__init__.py

```diff
@@ -90,11 +90,8 @@ def disable_signomials():
 from .variables import Variable, VectorVariable, ArrayVariable
 from .varkey import VarKey
 from .posyarray import PosyArray
-from .geometric_program import GeometricProgram
-from .signomial_program import SignomialProgram
```

Comment:
Do you still think (after reading the shortcuts file) that we should reimport these?